src/client/index.tsx

```diff
@@ -51,11 +51,7 @@ function convertVSCodeOutputToExecutResultOrDisplayData(
 ): nbformat.IExecuteResult | nbformat.IDisplayData {
     const isImage = cellInfo.mime.toLowerCase().startsWith('image/');
     const isJson = cellInfo.mime.toLowerCase().includes('json');
```

Comment:
I thought the isJson function was wrong?

Reply:
Didn't it need to check for actual json? Although I think that's a separate bug.

Reply:
yes thats wrong, waiting on @IanMatthewHuff not sure if he's working on a fix for that or not.
This issue fixes the image rendering issue.